### PR TITLE
I416 expand sensitivity

### PR DESCRIPTION
**Pull request by JIMENOFONSECA**
To try this. Run the next tests:
1. METHOD=sobol, N=1000, TEMPORAL_SCALE=yearly

2. METHOD=sobol, N=1000, TEMPORAL_SCALE=monthly

Six files should be created avery time you run each test in the location SAMPLES_FOLDER +'//'+RESULTS/

if you run the two tests. you should have 12 files in that folder.

The analysis part of the test No.2. can take some hours.... while the other one will take about 30 min.

### CODE REVIEW

**Comment by daren-thomas** [comment]
I get the following error for  `N=1000,  METHOD=sobol, TEMPORAL_SCALE=yearly`:

```
[darthoma@euler06 CEAforArcGIS]$ ls -l /cluster/scratch/darthoma/samples_sobol_1000/results
total 0
[darthoma@euler06 CEAforArcGIS]$ sh euler/analyze-simulations.sh
Traceback (most recent call last):
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/cluster/home/darthoma/CEAforArcGIS/cea/analysis/sensitivity/sensitivity_demand_analyze.py", line 209, in <module>
    analyze_sensitivity(samples_path=args.samples_folder, temporal_scale=args.temporal_scale)
  File "/cluster/home/darthoma/CEAforArcGIS/cea/analysis/sensitivity/sensitivity_demand_analyze.py", line 101, in analyze_sensitivity
    simulation_results = read_results(samples_path, samples_count, output_parameter, temporal_scale, month)
  File "/cluster/home/darthoma/CEAforArcGIS/cea/analysis/sensitivity/sensitivity_demand_analyze.py", line 195, in read_results
    building in iterable_num_buildings]
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/io/parsers.py", line 474, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/io/parsers.py", line 250, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/io/parsers.py", line 566, in __init__
    self._make_engine(self.engine)
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/io/parsers.py", line 705, in _make_engine
    self._engine = CParserWrapper(self.f, **self.options)
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/io/parsers.py", line 1072, in __init__
    self._reader = _parser.TextReader(src, **kwds)
  File "pandas/parser.pyx", line 350, in pandas.parser.TextReader.__cinit__ (pandas/parser.c:3173)
  File "pandas/parser.pyx", line 594, in pandas.parser.TextReader._setup_parser_source (pandas/parser.c:5912)
IOError: File /cluster/scratch/darthoma/samples_sobol_1000/result.0.0.csv does not exist
[darthoma@euler06 CEAforArcGIS]$ echo $N $METHOD $TEMPORAL_SCALE
1000 sobol yearly
```

**Comment by daren-thomas** [comment]
Also, for `TEMPORAL_SCALE=monthly`, I get the same error:

```
[darthoma@euler06 CEAforArcGIS]$ sh euler/analyze-simulations.sh
Traceback (most recent call last):
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/cluster/home/darthoma/CEAforArcGIS/cea/analysis/sensitivity/sensitivity_demand_analyze.py", line 209, in <module>
    analyze_sensitivity(samples_path=args.samples_folder, temporal_scale=args.temporal_scale)
  File "/cluster/home/darthoma/CEAforArcGIS/cea/analysis/sensitivity/sensitivity_demand_analyze.py", line 101, in analyze_sensitivity
    simulation_results = read_results(samples_path, samples_count, output_parameter, temporal_scale, month)
  File "/cluster/home/darthoma/CEAforArcGIS/cea/analysis/sensitivity/sensitivity_demand_analyze.py", line 195, in read_results
    building in iterable_num_buildings]
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/io/parsers.py", line 474, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/io/parsers.py", line 250, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/io/parsers.py", line 566, in __init__
    self._make_engine(self.engine)
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/io/parsers.py", line 705, in _make_engine
    self._engine = CParserWrapper(self.f, **self.options)
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/io/parsers.py", line 1072, in __init__
    self._reader = _parser.TextReader(src, **kwds)
  File "pandas/parser.pyx", line 350, in pandas.parser.TextReader.__cinit__ (pandas/parser.c:3173)
  File "pandas/parser.pyx", line 594, in pandas.parser.TextReader._setup_parser_source (pandas/parser.c:5912)
IOError: File /cluster/scratch/darthoma/samples_sobol_1000/result.0.0.csv does not exist
[darthoma@euler06 CEAforArcGIS]$ echo $N $METHOD $TEMPORAL_SCALE
1000 sobol monthly
[darthoma@euler06 CEAforArcGIS]$ ls -l /cluster/scratch/darthoma/
samples_morris_1/    samples_morris_11/   samples_morris_13/   samples_sobol_1/     samples_sobol_1000/  samples_sobol_1001/  __USAGE_RULES__
samples_morris_10/   samples_morris_12/   samples_morris_3/    samples_sobol_100/   samples_sobol_10000/ samples_sobol_3/
[darthoma@euler06 CEAforArcGIS]$ ls -l /cluster/scratch/darthoma/samples_sobol_1000/result
Display all 24001 possibilities? (y or n)
[darthoma@euler06 CEAforArcGIS]$ ls -l /cluster/scratch/darthoma/samples_sobol_1000/results
total **0**
```

File: cea/analysis/sensitivity/sensitivity_demand_analyze.py [hunk 1]
```diff
@@ -73,21 +59,57 @@ def analyze_sensitivity(samples_path):
...
+
+            # clear some memory:
+            simulation_results = None
+            # write out a worksheet for each analysis result (e.g. 'S1', 'ST', 'ST_conf' for method == 'sobol')
+            for analysis_variable in analysis_variables:
+                worksheet_name = analysis_variable
+                building_results = [result[analysis_variable] for result in analysis_results]
+                pd.DataFrame(building_results, columns=problem['names']).to_excel(writer, worksheet_name)
+            writer.save()
+        else:
```
Comment (JIMENOFONSECA) [inline cea/analysis/sensitivity/sensitivity_demand_analyze.py]: i changed the order of the if statement here

File: cea/analysis/sensitivity/sensitivity_demand_analyze.py [hunk 2]
```diff
@@ -73,21 +59,57 @@ def analyze_sensitivity(samples_path):
...
-            building_results = [result[analysis_variable] for result in analysis_results]
-            pd.DataFrame(building_results, columns=problem['names']).to_excel(writer, worksheet_name)
-    writer.save()
+        # create excel writer
+        folder = os.path.join(samples_path, 'results')
+        if not os.path.exists(folder):
+            os.makedirs(folder)
+
+        if temporal_scale is 'yearly':
+            writer = pd.ExcelWriter(
```
Comment (JIMENOFONSECA) [inline cea/analysis/sensitivity/sensitivity_demand_analyze.py]: not sure how to implement this, could this be an issue of refactoring for the future?

File: cea/analysis/sensitivity/sensitivity_demand_analyze.py [hunk 3]
```diff
@@ -142,22 +164,36 @@ def read_results(samples_folder, samples_count):
...
     --------------------
 
     - `$samples_folder/result.$i.csv` for i in range(samples_count)
     """
-    results = []
-    for i in range(samples_count):
-        result_file = os.path.join(samples_folder, 'result.%i.csv' % i)
-        df = pd.read_csv(result_file)
-        results.append(df)
+    iterable_samples_count = range(samples_count)
```
Comment (JIMENOFONSECA) [inline cea/analysis/sensitivity/sensitivity_demand_analyze.py]: I thought it was iterable of the form [0......n]. the variable is a vector not an integer.

File: cea/analysis/sensitivity/sensitivity_demand_analyze.py [hunk 4]
```diff
@@ -142,22 +164,36 @@ def read_results(samples_folder, samples_count):
...
+    if temporal_scale is 'yearly':
+        results = np.array(
+            [pd.read_csv(os.path.join(samples_folder, 'result.%i.csv' % item))[output_parameter].values for item in
```
Comment (JIMENOFONSECA) [inline cea/analysis/sensitivity/sensitivity_demand_analyze.py]: fixed, thanks :-)

File: cea/analysis/sensitivity/sensitivity_demand_analyze.py [hunk 5]
```diff
@@ -167,5 +203,7 @@ def read_results(samples_folder, samples_count):
     parser = argparse.ArgumentParser()
     parser.add_argument('-S', '--samples-folder', default='.',
                         help='folder to place the output files (samples.npy, problem.pickle) in')
+    parser.add_argument('-t', '--temporal-scale', default='yearly',
```
Comment (JIMENOFONSECA) [inline cea/analysis/sensitivity/sensitivity_demand_analyze.py]: fixed, i left the default argument too.

File: cea/analysis/sensitivity/sensitivity_demand_samples.py [hunk 6]
```diff
@@ -138,7 +134,7 @@ def sampler(method, problem, num_samples, sampler_parameters):
 
     parser = argparse.ArgumentParser()
     parser.add_argument('-m', '--method', help='Method to use valid values: "morris" (default), "sobol"',
```
Comment (JIMENOFONSECA) [inline cea/analysis/sensitivity/sensitivity_demand_samples.py]: fixed.

File: cea/analysis/sensitivity/sensitivity_demand_samples.py [hunk 7]
```diff
@@ -149,7 +145,7 @@ def sampler(method, problem, num_samples, sampler_parameters):
                         default=4)
     parser.add_argument('-S', '--samples-folder', default='.',
                         help='folder to place the output files (samples.npy, problem.pickle) in')
-    parser.add_argument('-V', '--variable-groups', default=['THERMAL'], nargs='+',
+    parser.add_argument('-V', '--variable-groups', default=['THERMAL', 'ARCHITECTURE', 'INDOOR_COMFORT', 'INTERNAL_LOADS'], nargs='+',
```
Comment (JIMENOFONSECA) [inline cea/analysis/sensitivity/sensitivity_demand_samples.py]: mm this seems challenging, would you give it a try? i do not want to break it

File: cea/analysis/sensitivity/sensitivity_demand_simulate.py [hunk 8]
```diff
@@ -82,16 +82,12 @@ def apply_sample_parameters(sample_index, samples_path, scenario_path, simulatio
     except IndexError:
         return None
 
-    # FIXME: add other variable groups here
     prop_thermal = Gdf.from_file(locator.get_building_thermal()).set_index('Name')
     prop_overrides = pd.DataFrame(index=prop_thermal.index)
     for i, key in enumerate(problem['names']):
         print("Setting prop_overrides['%s'] to %s" % (key, sample[i]))
         prop_overrides[key] = sample[i]
-        # prop_occupancy_df[key] = value
```
Comment (JIMENOFONSECA) [inline cea/analysis/sensitivity/sensitivity_demand_simulate.py]: :-)

File: cea/demand/demand_writers.py [hunk 9]
```diff
@@ -60,15 +62,16 @@ def results_to_csv(self, tsd, bpr, locator, date, building_name):
...
             keys = data.keys()
             columns = ['Name', 'Af_m2', 'Aroof_m2', 'GFA_m2', 'people0']
             columns.extend(keys)
 
             # add other default elements
             data.update({'Name': building_name, 'Af_m2': bpr.rc_model['Af'], 'Aroof_m2': bpr.rc_model['Aroof'],
                          'GFA_m2': bpr.rc_model['GFA_m2'], 'people0': tsd['people'].max()})
-            yearly_data = pd.DataFrame(data, index=[0]).to_csv(
+
+            # safe to disc
```
Comment (JIMENOFONSECA) [inline cea/demand/demand_writers.py]: fixed

File: cea/demand/demand_writers.py [hunk 10]
```diff
@@ -78,22 +81,72 @@ def write_to_csv(self, building_name, columns, hourly_data, locator):
...
+
     def __init__(self, gv):
         super(HourlyDemandWriter, self).__init__(gv)
 
     def write_to_csv(self, building_name, columns, hourly_data, locator):
         hourly_data.to_csv(locator.get_demand_results_file(building_name), columns=columns, float_format=FLOAT_FORMAT)
 
+    def write_totals_csv(self, building_properties, locator):
+        """read in the temporary results files and append them to the Totals.csv file."""
+        counter = 0
```
Comment (JIMENOFONSECA) [inline cea/demand/demand_writers.py]: fixed df=None, cool trick btw,

File: cea/demand/demand_writers.py [hunk 11]
```diff
@@ -78,22 +81,72 @@ def write_to_csv(self, building_name, columns, hourly_data, locator):
...
+        peaks = peaks.rename(
+            columns=dict((x + '_kWh', x + '0_kW') for x in self.vars_to_print[LOAD_VARS]))
+
+        monthly_data_new = monthly_data.merge(peaks, left_index=True, right_index=True)
+
+        monthly_data_new['Name'] = building_name
+        monthly_data_new.to_csv(locator.get_demand_results_file(building_name), index=False, float_format=FLOAT_FORMAT)
+
+
+    def write_totals_csv(self, building_properties, locator):
```
Comment (JIMENOFONSECA) [inline cea/demand/demand_writers.py]: Yes, we can add it. But i do not understand how the superclass work, is there another simpler structure for this?

File: cea/demand/demand_writers.py [hunk 12]
```diff
@@ -78,22 +81,72 @@ def write_to_csv(self, building_name, columns, hourly_data, locator):
...
+        """read in the temporary results files and append them to the Totals.csv file."""
+        counter = 0
```
Comment (JIMENOFONSECA) [inline cea/demand/demand_writers.py]: see above

File: cea/globalvar.py [hunk 13]
```diff
@@ -19,13 +19,13 @@
...
     def __init__(self):
-        self.scenario_reference = r'c:\reference-case-zug\baseline'
+        self.scenario_reference = r'c:\reference-case-open\baseline'
         self.print_partial = 'hourly' # hourly or monthly for the deamnd script
         self.print_yearly = True # print yearly values
         self.print_yearly_peak = True # print peak values
         self.date_start = '2016-01-01'  # format: yyyy-mm-dd
         self.seasonhours = [3216, 6192]
-        self.multiprocessing = True  # use multiprocessing / parallel execution if possible
+        self.multiprocessing = False  # use multiprocessing / parallel execution if possible
```
Comment (JIMENOFONSECA) [inline cea/globalvar.py]: i totally agree. I think we should add it to every script that has multiprocessing enabled. I believe the plot demands has it too...

**Comment by JIMENOFONSECA** [comment]
@daren-thomas, about the errors that are happening to you, did you run the other scripts first too? I made some changes that create a new pickle file of the output_parameters, so we do not have to refer to them later again.

File: cea/analysis/sensitivity/sensitivity_demand_samples.py (hunk 7 above)
Comment (daren-thomas) [inline cea/analysis/sensitivity/sensitivity_demand_samples.py]: yep. created issue  #436

File: cea/globalvar.py (hunk 13 above)
Comment (daren-thomas) [inline cea/globalvar.py]: Created issue #437

**Comment by daren-thomas** [comment]
It might have been a problem with the reference-case: I am updating that and running the simulations again.

**Comment by daren-thomas** [comment]
I updated the reference case and it still didn't work. Here is the output of running the demand script standalone (it works fine on my windows machine!)

```
[darthoma@euler10 CEAforArcGIS]$ python -m cea.demand.demand_main -s /cluster/home/darthoma/cea-reference-case/reference-case-open/baseline/ -w cea/ databases/CH/Weather/Zug.epw
failed to import from calc_tm.pyd, falling back to pure python functions
Running demand calculation for scenario /cluster/home/darthoma/cea-reference-case/reference-case-open/baseline/
Running demand calculation with weather file cea/databases/CH/Weather/Zug.epw
read input files
Traceback (most recent call last):
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/cluster/home/darthoma/CEAforArcGIS/cea/demand/demand_main.py", line 172, in <module>
    run_as_script(scenario_path=args.scenario, weather_path=args.weather)
  File "/cluster/home/darthoma/CEAforArcGIS/cea/demand/demand_main.py", line 161, in run_as_script
    demand_calculation(locator=locator, weather_path=weather_path, gv=gv)
  File "/cluster/home/darthoma/CEAforArcGIS/cea/demand/demand_main.py", line 92, in demand_calculation
    building_properties = BuildingProperties(locator, gv)
  File "cea/demand/thermal_loads.py", line 774, in __init__
    solar = get_prop_solar(locator).set_index('Name')
  File "cea/demand/thermal_loads.py", line 1260, in get_prop_solar
    solar = pd.read_csv(locator.get_radiation()).set_index('Name')
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/core/frame.py", line 2601, in set_index
    level = frame[col].values
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/core/frame.py", line 1791, in __getitem__
    return self._getitem_column(key)
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/core/frame.py", line 1798, in _getitem_column
    return self._get_item_cache(key)
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/core/generic.py", line 1084, in _get_item_cache
    values = self._data.get(item)
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/core/internals.py", line 2851, in get
    loc = self.items.get_loc(item)
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/core/index.py", line 1578, in get_loc
    return self._engine.get_loc(_values_from_object(key))
  File "pandas/index.pyx", line 134, in pandas.index.IndexEngine.get_loc (pandas/index.c:3824)
  File "pandas/index.pyx", line 154, in pandas.index.IndexEngine.get_loc (pandas/index.c:3704)
  File "pandas/hashtable.pyx", line 697, in pandas.hashtable.PyObjectHashTable.get_item (pandas/hashtable.c:12349)
  File "pandas/hashtable.pyx", line 705, in pandas.hashtable.PyObjectHashTable.get_item (pandas/hashtable.c:12300)
KeyError: 'Name'
```

@JIMENOFONSECA any ideas?

**Comment by JIMENOFONSECA** [comment]
Mmmm weird... i will check tomorrow when i get to the office

Best, Jimeno

On 14 Dec 2016, at 18:19, Daren Thomas <notifications@github.com<mailto:notifications@github.com>> wrote:


I updated the reference case and it still didn't work. Here is the output of running the demand script standalone (it works fine on my windows machine!)

[darthoma@euler10 CEAforArcGIS]$ python -m cea.demand.demand_main -s /cluster/home/darthoma/cea-reference-case/reference-case-open/baseline/ -w cea/ databases/CH/Weather/Zug.epw
failed to import from calc_tm.pyd, falling back to pure python functions
Running demand calculation for scenario /cluster/home/darthoma/cea-reference-case/reference-case-open/baseline/
Running demand calculation with weather file cea/databases/CH/Weather/Zug.epw
read input files
Traceback (most recent call last):
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/cluster/home/darthoma/CEAforArcGIS/cea/demand/demand_main.py", line 172, in <module>
    run_as_script(scenario_path=args.scenario, weather_path=args.weather)
  File "/cluster/home/darthoma/CEAforArcGIS/cea/demand/demand_main.py", line 161, in run_as_script
    demand_calculation(locator=locator, weather_path=weather_path, gv=gv)
  File "/cluster/home/darthoma/CEAforArcGIS/cea/demand/demand_main.py", line 92, in demand_calculation
    building_properties = BuildingProperties(locator, gv)
  File "cea/demand/thermal_loads.py", line 774, in __init__
    solar = get_prop_solar(locator).set_index('Name')
  File "cea/demand/thermal_loads.py", line 1260, in get_prop_solar
    solar = pd.read_csv(locator.get_radiation()).set_index('Name')
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/core/frame.py", line 2601, in set_index
    level = frame[col].values
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/core/frame.py", line 1791, in __getitem__
    return self._getitem_column(key)
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/core/frame.py", line 1798, in _getitem_column
    return self._get_item_cache(key)
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/core/generic.py", line 1084, in _get_item_cache
    values = self._data.get(item)
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/core/internals.py", line 2851, in get
    loc = self.items.get_loc(item)
  File "/cluster/apps/python/2.7.6/x86_64/lib64/python2.7/site-packages/pandas-0.16.1-py2.7-linux-x86_64.egg/pandas/core/index.py", line 1578, in get_loc
    return self._engine.get_loc(_values_from_object(key))
  File "pandas/index.pyx", line 134, in pandas.index.IndexEngine.get_loc (pandas/index.c:3824)
  File "pandas/index.pyx", line 154, in pandas.index.IndexEngine.get_loc (pandas/index.c:3704)
  File "pandas/hashtable.pyx", line 697, in pandas.hashtable.PyObjectHashTable.get_item (pandas/hashtable.c:12349)
  File "pandas/hashtable.pyx", line 705, in pandas.hashtable.PyObjectHashTable.get_item (pandas/hashtable.c:12300)
KeyError: 'Name'


@JIMENOFONSECA<https://github.com/JIMENOFONSECA> any ideas?

**Comment by daren-thomas** [comment]
found the problem! (github lfs... i need to manually upload the radiation data. i'll write back once i've done testing)

**Comment by JIMENOFONSECA** [comment]
@daren-thomas did it work now?

**Comment by daren-thomas** [comment]
I got "disk quota exceeded" and an email from the euler admins... I am cleaning up my scratch folder as I type this and will have to run the simulations again to test. Also, I think we should make a new plan for the results.csv files... I think that can be a separate issue though.

**Comment by daren-thomas** [comment]
I am not able to produce a difference with the TEMPORAL_SCALE parameters. The end result is always just an Excel file for each output variable analyzed and a single value per building and input parameter. Am I missing something? Shouldn't there be monthly output?

**Comment by daren-thomas** [comment]
As far as I can tell this is now fully functional. There are some things that could be improved incrementally and issues have been created to address this.

**Pull request merged**